src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -364,6 +364,16 @@ static String validateConfig(Map<String, String> config) {
       if (!BufferThreshold.validateBufferThreshold(config, IngestionMethodConfig.SNOWPIPE)) {
         configIsValid = false;
       }
+      if (config.containsKey(SnowflakeSinkConnectorConfig.SCHEMATIZATION_ENABLE_CONFIG)) {
```

Comment:
did you mean to check `snowpipe_streaming` here?

Reply:
This check is to make sure snowpipe & enable.schematization are not set at the same time.

Reply:
nvm, didnt see the above if case. Thanks